dayrepo/snippets/views.py

```diff
@@ -122,7 +122,7 @@ def post(self, request):
         )
         duties_trouble.save()
 
-        # Process form
+        # Process form　工程テーブルフォーム保存
```

Comment:
```suggestion
        # 工程テーブルフォーム保存
```
英語と日本語で意味を重複させる意味はない